engine/hunspell_table.py

```
@@ -67,11 +68,17 @@
 import itb_version
 
+itb_ollama: Optional[ModuleType]
+_itb_ollama_import_error: Optional[Exception]
+if TYPE_CHECKING:
+    from itb_ollama import ItbOllamaClient
 try:
     import itb_ollama
-    IMPORT_ITB_OLLAMA_ERROR = None
-except ImportError as error:
-    IMPORT_ITB_OLLAMA_ERROR = error
+except Exception as error:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)